guides/doc-Release_Notes/topics/katello-contributors.adoc

```diff
@@ -1,24 +1,24 @@
-Adam Růžička
-Amir Fefer
-Bernhard Suttner
-Chris Roberts
-Evgeni Golov
-Ewoud Kohl van Wijngaarden
-Girija Soni
-Hao Yu
-Ian Ballou
-Jeremy Lenz
-Karolína Małyjurková
-Leoš Stejskal
-Lucy Fu
-Lukas Magauer
-Maria Agaphontzev
-Markus Bucher
-Nagoor Shaik
-Partha Aji
-Quinn James
-Samir Jha
-Sayan Das
-Stephan Bergmann
-Trevor Allison
+Adam Růžička,
```

Comment:
commas added to match the formatting of foreman-contributors

Reply:
Oh, I like this. It's much easier to maintain than a huge sentence and I didn't think of this. We should standardize on this.